coderd/database/pubsub/pubsub.go

```diff
@@ -307,7 +319,13 @@ func (p *pgPubsub) listen() {
 	}
 }
 
-func (p *pgPubsub) listenReceive(notif *pq.Notification) {
+func (p *PGPubsub) listenReceive(notif *pq.Notification) {
+	sizeLabel := messageSizeNormal
+	if len(notif.Extra) >= colossalThreshold {
+		sizeLabel = messageSizeColossal
+	}
```

Comment:
Might be interesting to see a "high" label as well, say at 30-50% of the limit. If we're pushing a lot of data here, it could affect pg performance.

Reply:
If we're interested in more granular sizes, maybe we should make this metric a histogram instead.

Reply:
How about we also total up the number of bytes transmitted and received